src/main/java/uno/fastcampus/testdata/dto/TableSchemaDto.java

```diff
@@ -44,7 +44,7 @@ public static TableSchemaDto fromEntity(TableSchema entity) {
     }
 
     public TableSchema createEntity() {
-        TableSchema entity = TableSchema.of(this.schemaName(), this.userId());
+        TableSchema entity = TableSchema.of(schemaName, userId);
```

Comment:
`schemaName` 이거 이름을 `name`으로 가는 건 어때요?

```suggestion
        TableSchema entity = TableSchema.of(name, userId);
```

Reply:
응 싫어요